src/cf-expandables/src/ExpandableTransition.js

```diff
@@ -41,25 +40,28 @@ function ExpandableTransition( element, classes ) {
       _transitionCompleteBinded
     );
 
-    if ( contains( element, classObject.OPEN_DEFAULT ) ) {
-      _baseTransition.applyClass( classObject.EXPANDED );
-      element.style.maxHeight = element.scrollHeight + 'px';
+    if ( contains( element, CLASSES.OPEN_DEFAULT ) ) {
+      this.expand();
     } else {
-      previousHeight = element.scrollHeight;
-      _baseTransition.applyClass( classObject.COLLAPSED );
+      this.collapse();
     }
 
     return this;
   }
 
+  /* istanbul ignore next */
```

Comment:
Why's this needed?

Reply:
Because it's impossible to test the transitionEnd in a testing env.